net-p2p/rtorrent/files/rtorrent-0.10.0-lto.patch

```diff
@@ -0,0 +1,89 @@
+https://bugs.gentoo.org/861848
+https://github.com/rakshasa/rtorrent/issues/1264
+https://github.com/rakshasa/rtorrent/pull/1291
+https://github.com/rakshasa/rtorrent/commit/c1d076f1822752d4956a884a9aa9e6b0026b11f3
+https://github.com/rakshasa/rtorrent/commit/38b39bdafc9edd7e7e72672e6fcbd397b6c2cab8
+
+From c1d076f1822752d4956a884a9aa9e6b0026b11f3 Mon Sep 17 00:00:00 2001
+From: stickz <stickman002@mail.com>
+Date: Sun, 29 Sep 2024 13:24:01 -0400
+Subject: [PATCH] Fixing static building
+
+@rakshasa A major hosting provider with rTorrent has mentioned the following revision is required for static builds to pass.
+---
+ configure.ac | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/configure.ac b/configure.ac
+index ca7d9ca..a4dd61d 100755
+--- a/configure.ac
++++ b/configure.ac
+@@ -11,6 +11,7 @@ AC_DEFINE([API_VERSION], [10], [api version])
+ AC_PROG_RANLIB
+ AC_PROG_CXX
+ AC_SYS_LARGEFILE
++AC_PROG_LIBTOOL
+ 
+ AX_CXX_COMPILE_STDCXX([14], [noext], [mandatory])
+ 
+-- 
+2.45.2
+
+From 38b39bdafc9edd7e7e72672e6fcbd397b6c2cab8 Mon Sep 17 00:00:00 2001
+From: Adam Sampson <ats@offog.org>
+Date: Sun, 29 Sep 2024 22:14:16 +0100
+Subject: [PATCH] Add ax_require_defined.m4 to scripts.
```

Comment:
This is just a vendored copy of dev-build/autoconf-archive anyway. It is used when running `eautoreconf`, which we admittedly do.